### PR TITLE
feat: create review model-#26

### PR DESCRIPTION
**Pull request by chloe630**
feat: create review model #26

### CODE REVIEW

File: db/models/review.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
+'use strict'
+const Sequelize = require('sequelize')
+const db = require('./index.js')
```
Comment (JeffKandel) [inline db/models/review.js]: db is not required here.  In line 5 'db' is just a parameter passed to the function.  You can remove line 3 completely.

File: db/models/review.js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  content: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
   num_stars: {
-    type: FLOAT,
+    type: Sequelize.DECIMAL(1, 2),
```
Comment (JeffKandel) [inline db/models/review.js]: Hi Chloe, we made a mistake here!

Reviews should be type Sequelize.INTEGER.

Each review has only 1,2,3,4, or 5 stars.

Then the product model is where we average things.  

Can you update this and the test?

**Review by JeffKandel — CHANGES_REQUESTED** [review]
Hi Chloe, please see my notes below about the integer values of an individual review.

File: db/models/review.js (hunk 1 above)
Comment (chloe630) [reply db/models/review.js]: oops I will delete them!

File: db/models/review.js (hunk 2 above)
Comment (chloe630) [reply db/models/review.js]: Hi Jeff, yes the test was what confused me when I was writing this file. 
Sure the review should take only 1-5stars, I'll modify the num_star type in this model and the review test too.

File: db/models/review.js [hunk 3]
```diff
@@ -1,7 +1,3 @@
-'use strict'
-const Sequelize = require('sequelize')
-const db = require('./index.js')
```
Comment (JeffKandel) [inline db/models/review.js]: Lines 1 and 2 are still necessary or there will be errors, it's just db that isn't used...

**Review by JeffKandel — CHANGES_REQUESTED** [review]
Bring back the sequelize require and use strict and then this will be good to go!

**Review by JeffKandel — APPROVED** [review]
Please merge this!

**Pull request merged**